database/src/scylladb/mod.rs

```diff
@@ -0,0 +1,550 @@
+pub mod rpc_server;
+pub mod state_indexer;
+pub mod tx_indexer;
+
+use scylla::prepared_statement::PreparedStatement;
+use scylla::retry_policy::{QueryInfo, RetryDecision};
+use scylla::transport::errors::QueryError;
+
+/// Additional database options for scylla database
+///
+/// `preferred_dc: Option<String>`,
+/// ScyllaDB preferred DataCenter
+/// Accepts the DC name of the ScyllaDB to filter the connection to that DC only (preferrably).
+/// If you connect to multi-DC cluter, you might experience big latencies while working with the DB.
+/// This is due to the fact that ScyllaDB driver tries to connect to any of the nodes in the cluster
+/// disregarding of the location of the DC. This option allows to filter the connection to the DC you need.
+/// Example: "DC1" where DC1 is located in the same region as the application.
+///
+/// `keepalive_interval: Option<u64>`
+/// ScyllaDB keepalive interval
+///
+/// `max_retry: u8`
+/// Max retry count for ScyllaDB if `strict_mode` is `false`
+///
+/// `strict_mode: bool`
+/// Attempts to store data in the database should be infinite to ensure no data is missing.
+/// Disable it to perform a limited write attempts (`max_retry`)
+/// before skipping giving up and moving to the next piece of data
+#[cfg(feature = "scylla_db")]
+pub struct AdditionalDatabaseOptions {
+    pub preferred_dc: Option<String>,
+    pub keepalive_interval: Option<u64>,
+    pub max_retry: u8,
+    pub strict_mode: bool,
+}
+
+// pub(crate) struct MyScyllaManager {
```

Comment:
What is the purpose of this comment here? Is it a leftover you need to clean up?